lib/config_upgrader.js

```diff
@@ -235,8 +236,9 @@ class ConfigUpgrader {
       const config = new Config({
         configFile: configFile,
         cwd: process.cwd()
-      });
-      upgrader.upgrade(config.useSpecificConfig);
```

Comment:
Agree, it is confusing. Closing brace is 2 spaces outdented. In that it is correct. But the second argument is level with outer function call. In that it is incorrect. Probably, have to move the opening brace to a new line and indent all arguments.